### ISSUE TITLE
Page and PageController show up in IDEs as valid classes

### ISSUE THREAD

**Issue by dhensby** [silverstripe/recipe-cms#9]
When using an IDE's "find class" feature, Page and PageController will often match two different files (the one in the project and the one in recipe-cms).

Please can we rename the files to have an extension like `.tmpl` or whatnot so that IDEs don't recognise them as PHP classes.

When we copy the files into projects we can copy and rename them instead to just copying them.

**Comment by chillu** [comment]
+1 - wanna send a PR?

**Comment by dhensby** [comment]
OK - so we need a PR to [silverstripe/recipe-plugin](https://github.com/dhensby/recipe-plugin) to strip the `.tmpl` extension off files first.

**Comment by tractorcow** [comment]
Yep thanks for your work on this @dhensby I like the idea and think it's working well. Only one last thing on https://github.com/silverstripe/recipe-plugin/pull/5 to ensure the base project is renamed and we're golden.
